pkg/tsdb/influxdb/response_parser.go

```diff
@@ -169,21 +169,21 @@ func transformRows(rows []Row, query Query) data.Frames {
 
 				name := string(formatFrameName(row, column, query, frameName[:]))
 
-				timeField := data.NewField("time", nil, timeArray)
+				timeField := data.NewField("Time", nil, timeArray)
```

Comment:
If I understand correclty, we rollback to previous capitalization. So we will break dashboards created with the current version right (with lowercase)?